.github/workflows/triage_add_to_project.yml

```diff
@@ -40,11 +42,12 @@ jobs:
 
           echo 'IS_COLLABORATOR='$(jq -r '.data.repository.collaborators.totalCount' collaborators.json) >> $GITHUB_ENV
       - uses: actions/add-to-project@v0.4.1
-        # only add issues/prs from outside contributors to the project
         if: ${{ env.IS_COLLABORATOR == 0 || github.event.repository.name == 'cypress-support-internal' || github.event.pull_request.user.login == 'github-actions[bot]' || github.event.issue.user.login == 'github-actions[bot]' }}
         with:
           project-url: https://github.com/orgs/${{github.repository_owner}}/projects/${{env.PROJECT_NUMBER}}
           github-token: ${{ secrets.ADD_TO_TRIAGE_BOARD_TOKEN }}
+          labeled: "type: feature", "type: enhancement"          
```

Comment:
Unfortunately, I doubt this will have much impact since it would require someone to add the labels during PR creation which I don't think many people will be doing.

Reply:
We could create another action that removes it from the board when we add the label.